pages/product.html

```diff
@@ -0,0 +1,430 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>nftcart | products</title>
+
+    <!-- Favicon -->
+    <link rel="icon" type="image/x-icon" href="../assets/nft.ico" />
+
+    <!-- Fontawesome -->
+    <script
+      src="https://kit.fontawesome.com/b9c207240a.js"
+      crossorigin="anonymous"></script>
```

Comment:
You can consider moving font-awesome script to main css, so that you don't have to replicate it to all of your html files

Reply:
Yes, this just for momentary basis.